### PR TITLE
Fontconfig

### PR DESCRIPTION
**Pull request by patriciogonzalezvivo**
Solving fontconfig problem by compiling freetype 2.6.12 on the amazon GPU AMI server!!

### CODE REVIEW

File: paparazzi.sh [hunk 1]
```diff
@@ -144,8 +144,15 @@ case "$1" in
         ./configure --prefix=/usr
         make -j $N_CORES
         sudo make install
+        # Do some linking
+        sudo rm /usr/lib64/libfreetype.so
```
Comment (tallytalwar) [inline paparazzi.sh:148]: maybe sure to let the user know in the readme what giving sudo permission is here!

Comment (patriciogonzalezvivo) [reply paparazzi.sh:148]: hehehe... true! the GCC 4.9 need similar linking : S... @rmarianski will fix this when he makes RPM packages for it https://github.com/tangrams/paparazzi/issues/17. This is more like a log track to communicate back and forth with @rmarianski

Comment (tallytalwar) [reply paparazzi.sh:148]: Sure, I have previously mentioned how this shell script is great but at the same time very risky with the kind of sudo privileges this gets. :D. Good luck.

Comment (patriciogonzalezvivo) [reply paparazzi.sh:148]: is the MOST risky

Comment (patriciogonzalezvivo) [reply paparazzi.sh:148]: Paparazzi, living dangerously...

**Pull request merged**